pyproject.toml

```
@@ -45,7 +45,9 @@ Issues = "https://github.com/opendatahub-io/vllm-tgis-adapter/issues"
 Source = "https://github.com/opendatahub-io/vllm-tgis-adapter"
 
 [project.scripts]
-grpc_healthcheck = "vllm_tgis_adapter.healthcheck:cli"
+grpc-healthcheck = "vllm_tgis_adapter.healthcheck:cli"
```

Please do not change this:
```suggestion
grpc_healthcheck = "vllm_tgis_adapter.healthcheck:cli"
```